corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -21,7 +21,11 @@
 from casexml.apps.phone.checksum import Checksum, CaseStateHash
 import logging
 
-logger = logging.getLogger('phone.models')
+
+def _get_logger():
+    # for some strange reason if you define logger the normal way it gets silenced in tests.
+    # this is a hacky workaround to that.
+    return logging.getLogger(__name__)
```

Comment:
I spent what felt like a reasonable amount of time trying to find a reason why this was happening and eventually gave up.